src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -39,38 +39,44 @@ import {getPatientSurvivals} from "pages/resultsView/SurvivalStoreHelper";
 import {SURVIVAL_CHART_ATTRIBUTES} from "pages/resultsView/survival/SurvivalChart";
 import {COLORS} from "pages/studyView/StudyViewUtils";
 import {AlterationEnrichment, Group} from "../../shared/api/generated/CBioPortalAPIInternal";
-import ListIndexedMap from "shared/lib/ListIndexedMap";
+import ListIndexedMap, {ListIndexedSet} from "shared/lib/ListIndexedMap";
 import {GroupComparisonTab} from "./GroupComparisonPage";
 import {Session} from "../../shared/api/ComparisonGroupClient";
 import { calculateQValues } from "shared/lib/calculation/BenjaminiHochbergFDRCalculator";
+import {getStudiesAttr} from "./comparisonGroupManager/ComparisonGroupManagerUtils";
 
-export type GroupComparisonURLQuery = {
-    localGroups:string; // comma separated list
-
-    // OR
-
-    fromChart:string; //"true" or "false"
-    unshareableLocalKey:string;
-};
+export enum OverlapStrategy {
+    INCLUDE = "INCLUDE",
+    EXCLUDE = "EXCLUDE",
+    GROUP = "GROUP"
+}
 
 export default class GroupComparisonStore {
 
-    @observable currentTabId:GroupComparisonTab|undefined = undefined;
-    @observable excludeOverlapping:boolean = false;
-    @observable sessionId:string;
+    @observable private _currentTabId:GroupComparisonTab|undefined = undefined;
+    @observable private _overlapStrategy:OverlapStrategy = OverlapStrategy.INCLUDE;
+    @observable private sessionId:string;
```

Comment:
Seems should be `_sessionId`?

Reply:
since its just private and theres no setter, I feel like it makes sense to leave it like this